send-transaction-service/src/send_transaction_service.rs

```diff
@@ -115,6 +115,8 @@ pub struct Config {
     pub batch_send_rate_ms: u64,
     /// When the retry pool exceeds this max size, new transactions are dropped after their first broadcast attempt
     pub retry_pool_max_size: usize,
+    pub tpu_peers: Option<Vec<SocketAddr>>,
+    pub also_leader: bool,
```

Comment:
Do we need this new field? Not to be overly cute, but seems like we could set `leader_forward_count: 0` when tpu_peers is some but also_leader is false.

Reply:
ah yeah, good point i'll check whether `leader_forward_count` logic will play nice